scripts.js

```diff
@@ -0,0 +1,323 @@
+// 1
+const thingsInHome = ["mint", "basil", "cactus", "table", "wooden spoon", "bread"];
+const thingsInGarden = ["apple", "trees", "stairs", "plum", "wooden bench"];
+
+// -- edit below --
+const madeOutOfWood = [thingsInHome[3], thingsInHome[4], thingsInGarden[1], thingsInGarden[2], thingsInGarden[4]];
+const edibles = [thingsInHome[0], thingsInHome[1], thingsInHome[5], thingsInGarden[0], thingsInGarden[3]];
+// -- edit above --
+
+console.log(
+    `made out of wood: ${JSON.stringify(madeOutOfWood)}\n edibles: ${JSON.stringify(edibles)}`
+);
+
+// 2
+function getSmallestNumber(numbers) {
+  let smallestNumber = numbers[0];
+
+  for (let i = 1; i < numbers.length; ++i) {
+    const currentNumber = numbers[i];
+
+    if (currentNumber < smallestNumber){
+      smallestNumber = currentNumber;
+    }
+  }
+
+  return smallestNumber;
+}
+
+function getSmallestNumberSort(numbers) {
+  numbers.sort(function(a, b){return a - b});
```

Comment:
Don't use one-letter variables. Please avoid repeating the same mistake in multiple PRs